modules/core/server/services/email.server.service.js

```diff
@@ -128,7 +129,7 @@ exports.sendRemoveProfile = function (user, callback) {
   const campaign = 'remove-profile';
 
   const params = exports.addEmailBaseTemplateParams({
-    subject: 'Confirm removing your Trustroots profile',
+    subject: 'Confirm removing your ' + brandName + ' profile',
```

Comment:
Just a JS syntax tip: I really like using template strings compared to additions to put variables in strings. If you are unfamiliar with the syntax you can check out the Mozilla docs here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I tried and for some reason it did not work...